migrations/V118__Update_Precedence_Rules.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TYPE YES_NO_FILTER AS ENUM (
```

Comment:
Non blocking, but filter feels a bit misleading here. It's not really a filter, it's a question answer. Maybe `YES_NO_TYPE` would be more consistent?
```suggestion
CREATE TYPE YES_NO_TYPE AS ENUM (
```

Reply:
That makes sense. I was unsure of the naming for this one, I think your suggestion is a good one.